features/waterOrdering/replies.js

```diff
@@ -10,6 +10,14 @@ module.exports = {
     'Done :white_check_mark:.',
     'Yes!',
   ],
+  noRespect: [
+    'Maybe "Please"? :grumpy:',
+    'You ask me for a service, but do it without respect? :godfather:',
+    'Actually, I deserve respect :dude:',
+    'How dare you not respect me? :accepted:',
+    'Water shall not pass!!! :not_pass:',
+    'Demand respect I :yoda:',
```

Comment:
Can we somehow say that `please` is required in messages above?

Just not sure how to do better but would like people to understand that Mona is waiting for `please`